src/Entity/EntityId.php

```diff
@@ -17,6 +18,30 @@
 
 	protected $serialization;
 
+	const PATTERN = '/^:?(\w+:)*[^:]+\z/';
```

Comment:
Ah sorry, this is in EntityId. I would have expected the pattern for checking the prefix syntax to have a different constant name than the patterns that check the final parts of the ID.